aspnet-core/xml/ns-Microsoft.AspNetCore.Connections.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Connections">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains types and abstractions that support low level applications that work with the raw bytes from a connection.</summary>
```

Comment:
Sorry. I know you just applied my previous suggestions, but I accidently rewrote these summaries because 
https://github.com/dotnet/aspnetcore/issues/27286 assigned Microsoft.AspNetCore.Connections and Microsoft.AspNetCore.Http.Connections.Features to both of us. I thing we should point out that these namespaces are very specific to SignalR. The API docs don't currently make that very clear.

```suggestion
    <summary>Contains types for configuring SignalR connections.</summary>
```

Reply:
Isn't Http.Connections for SignalR and just Connections for Kestrel?

Reply:
You're totally right! My most recent suggestion are the summaries I'm planning to use for Microsoft.AspNetCore.*Http*.Connections and Microsoft.AspNetCore.Http.Features. I'm happy with these as-is then.